### PR TITLE
add documentation for nodes based shapes in map composer

### LINKED ISSUE #943: [FEATURE][composer] Add tools for drawing polygon and polylines
Original commit: https://github.com/qgis/QGIS/commit/7a8a5411e09a3076f96d727166d8877304b29543 by nyalldawson

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by pblottiere**
This PR add documentation for the new node-based shape tool in the map composer. I updated the image of the map composer window because of the new icons' tool.

Note that this feature is for 2.16.

Fix #943

### CODE REVIEW

**Comment by yjacolin** [comment]
@pblottiere could you add the Issue number somewhere in the PR ? Thanks.

File: source/docs/user_manual/print_composer/composer_items/composer_nodes_shape.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.. _figure_composer_nodes_shape:
+
+.. only:: html
+
+   **Figure Composer Nodes Shape:**
+
+.. figure:: /static/user_manual/print_composer/shape_nodes_properties.png
+   :align: center
+
+   Nodes Shape Item properties Tab |nix|
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_nodes_shape.rst]: @pblottiere we no longer put |nix|, |win|... at the end of figure title

File: source/docs/user_manual/print_composer/composer_items/index.rst [hunk 2]
```diff
@@ -21,4 +21,4 @@ Composer Items
    composer_html_frame
    composer_basic_shape
    composer_arrow
-
+   composer_nodes_shape
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/index.rst]: I not yet know how but, due to how short is the chapter and initiatives to have less split chapters (see https://github.com/qgis/QGIS-Documentation/pull/760#issuecomment-174222441), I think we'll better merge the chapter with another one (maybe with ``composer_basic_shape``?)

**Comment by DelazJ** [comment]
Thanks for documenting this feature!

**Comment by yjacolin** [comment]
@pblottiere could you rebase your PR from master? It is time to merge your contribution :)

**Comment by DelazJ** [comment]
I'd also be in favor of merging this chapter with http://docs.qgis.org/testing/en/docs/user_manual/print_composer/composer_items/composer_basic_shape.html and http://docs.qgis.org/testing/en/docs/user_manual/print_composer/composer_items/composer_arrow.html. They are all about adding new shapes in composer. Having a single page for that sounds coherent to me and reduce the number of files to build (note that the three chapters are really short)

**Pull request merged**